src/build_platform/local_docker.rs

```diff
@@ -74,6 +74,10 @@ impl LocalDocker {
             vec!["build"]
         };
 
+        let args = self.context.docker_build_options();
+        args.iter()
```

Comment:
maybe more readable if you turn it into a for loop.
for v in args.iter() {
   for string in v.iter() {
       docker_args.push(String::as_str(string))
    }
}